librocksdb-sys/build.rs

```diff
@@ -154,6 +154,12 @@ fn build_rocksdb() {
         config.define("OS_ANDROID", None);
         config.define("ROCKSDB_PLATFORM_POSIX", None);
         config.define("ROCKSDB_LIB_IO_POSIX", None);
+
+        if &target == "armv7-linux-androideabi" {
+            config.define("_FILE_OFFSET_BITS", Some("32"));
```

Comment:
it looks like the real fix is here? so can we keep this if, remove the else if `aarch64-linux-android` and on line 227 do:

```
if &target != "armv7-xxxx' && env::var("CARGO_CFG_TARGET_POINTER_WIDTH").unwrap() != "64" {
}
```

Reply:
Yep. We can only keep the change for the "armv7-linux-androideabi" target.